### ISSUE TITLE
authedmine friend/client is not valid? who is valid? you?

### ISSUE THREAD

**Issue by thEpisode** [easylist/easylist#712]
@ryanbr close #710 with "Forwarding your friends/clients to this issue report, doesn't make it valid. As stated, the filters are staying put." and locked the conversation? What the hell? what happend here? this is a social network to upload/download code, create community, and more; Github allows to make private repos if you have a "public" repo and allow issues, why block our comments? I'm a client and we are not getting rich minning with users consentiment because rate is so low but we have a bussines model on it and we want to pay to our employees with mining criptocurrencies, whats wrong with it? I'm not a friend of Coinhive devs (I want to develop and improve their company, really) but I'm a client and I want to monetize my blog and personal projects, if we aren't valid, who? you? your dog? your mother?

**Comment by thEpisode** [comment]
Get out from GitHub! or use and **pay** private repo if you approve GitHub's Business Model, stop to use free tools to block free tools.

**Comment by Leseratte10** [comment]
To the EasyPrivacy team: What else makes it valid? The last issue has been closed with the comment "please get examples where this is used", then we open a new one giving you 30+ actual examples, and you lock it again?

**Comment by enricodias** [comment]
EasyPrivacy team seems to have a bias against miners and are enforcing their opinion to others. This is a is discriminatory behavior and it's against GitHub's terms and conditions.

**Comment by ViolentlyPeaceful** [comment]
**advertisement** (ădˌvər-tīzˈmənt, ăd-vûrˈtĭs-, -tĭz-)►
n.	The act of advertising.
n.	A notice, such as a poster or a paid announcement in the print, broadcast, or electronic media, designed to attract public attention or patronage.

You do understand that cryptocurrency mining, with consent from the user, **is not advertisement**, right? Easylist is supposed to block intrusive advertisements. If an user wants to see the ads, it removes the list or disables the AdBlock.

If an user wants to mine cryptocurrencies **without seeing advertisements** they are unable, because you're not making the distinction between advertisement and cryptocurrency mining, which are very different things.

This is just common sense. No problems in assuming that it was a wrong decision and coming back on it. I hope you can see as clearly as us that the way you're behaving right now is edging on the promotion of censorship. I hope you can consider this a constructive criticism. Cheers.

**Comment by JimmyRecard** [comment]
You guys need to stop this.

While I think crypto mining with user consent shouldn't be blocked, the way you're going about this and bullying and spamming Fanboy and other EasyList maintainer is getting you nowhere.

Remember that EasyList is not accountable to you but to their users. They make a product for their users and your only way forward is to show that allowing authenticated mining in in their user's interest.

Instead you're organising a witch hunt, spamming GitHub and hardening maintainer's resolve against inbrowser mining.

Just stop. Take a breath. Stop turning list maintainers against this worthy alternative to advertising. If your product is truly the next big thing then EasyList will see that in due time.

**Comment by ViolentlyPeaceful** [comment]
Although I agree with what you said, I find it very hard "to show that allowing authenticated mining is in their user's interest." if it's already blocked. This is a very new technology that is being explored and tested. If you begin to block in its infancy, how it's supposed to flourish into new and interesting ways to monetize web traffic without advertisements?

I don't agree with witch hunting. It's bad. To be quite honest, I don't even use CoinHive or any other web browsing mining service. I'm just an advocate of cryptocurrencies in general and I find counter-productive to the scene to block something only because "you don't understand it."

If you do have suggestions on how people can prove EasyList maintainers that cryptocurrency mining with user consent shouldn't be blocked -- considering that no one can effectively use it because they're blocking it -- let us know.

**Comment by Leseratte10** [comment]
@JimmyRecard I understand that, and I won't open any more issues if this issue gets closed again. However, I will continue commenting on other open issues and try to convince the easylist operators to unblock it. 

@ViolentlyPeaceful That is exactly what I am saying. There is a new technique, and it is immediately blocked despite having nothing to do with advertisements, and the maintainers don't do anything. They asked for examples, they got examples, and just closed another issue. 

How can we "prove" that this shouldn't be blocked? We explained that there is no way to abuse this without consent, we gave multiple web sites using this technique and we explained that this could be a new way to support web pages without ads. What else should the users / website owners do?

**Comment by thEpisode** [comment]
@Leseratte10 I will create as many Issues as I can... sorry is spam but **NOT BLOCK THE COMMENTS!!** @enricodias says: "This is a is discriminatory behavior and it's against GitHub's terms and conditions."

**Comment by enricodias** [comment]
@JimmyRecard I'm a user and my last statement stands.

Also, any adblock using this list will fail in the chrome's single purpose policy and may be removed from the webstore. Web mining is not an ad. If users want to block miners they should install minerBlock of something similar.

**Comment by EnumC** [comment]
While I agree with everyone here, I also find some of these remarks to be inflammatory. These comments won't convince the maintainer to revoke the blacklist but do quite the opposite. Spamming Issue Reports is not a solution. I suggest the maintainer to keep cryptominers such as this one on a separate filter list (Especially ones that request user permissions), as these don't necessarily belong under ads per-say, as it also offer a captcha service which would be broken by universally blocking the entire domain. Of course, everyone can have a huge argument about whether it's in EasyList's place to block cryptominers, but an optional opt-in list I would think would be the best solution for the community. Adblock extension authors could optionally enable it by default, or allow the users to enable the filter.

**Comment by Leseratte10** [comment]
The reason why the community opened so many issue reports is just that they were all closed immediately. Luckily this issue has now been open for two days and not been closed so maybe it'll now be possible to have an actual conversation / discussion instead of having to complain about closing issues. 

The "close issue" function of github, in my opinion, should be used for topics / issues which are fixed, which means, the issue reporter and the project team agree with eachother. This means, if a page is reported to be added to a black list, it should be closed when it is either added to the black list, or the community and the maintainers agree on why it will not be added to a black list. Same for removing something. 

As you said, having just one issue would have reduced half of the spam comments because one would not need to repeat everything. 

Plus, the first two or even three issue reports contained no spam, everyone was friendly and just requested for this to be removed from the black list? What was the reaction of the team? Whitelist authedmine for one specific domain which uses it. Great. What about the others? How can you then blame the community from being a bit pissed about this?

I agree with what you said about another black list, and I already said that in preivous issues, too. EasyPrivacy, according to its name and its description, protects privacy and removes trackers and such. Not voluntary mining.

**Comment by lawl** [comment]
Uh what? So now I have to disable my adblocker on websites that explicitly don't want to show me ads?

**Comment by EnumC** [comment]
@lawl I think they are treating this just like ads. They block it by default, and you would have to whitelist the website you actually want to support. Makes sense for ones that do not ask the user whether they will allow it, but imo is redundant if it asks for permission anyways.

**Comment by lawl** [comment]
@EnumC yeah it makes complete sense if they mine without asking, but the entire point is here that they don't....

**Comment by golyalpha** [comment]
The whole point of authedmine.com (owned by coinhive.com) is to enforce user consent if a website wishes to mine with their computers.
EasyPrivacy list is marketed as an optional blocklist for blocking trackers. Authedmine.com doesn't track or store identifying information unless requested by the website using it (and even then, the website can generate a random token, assign it to a user internally on it's side, and provide the token to authedmine). 
On top of that, UBlock Origin enables EasyPrivacy by *default*(!!!), without asking the user at all, which kinda goes against the "optional blocklist" side of things.

Conclusion: By all means, block coinhive.com - it hosts the version of the miner that doesn't require user consent, which can silently mine in the background, using up system resources, without the permission of a user. Do not block authedmine.com as it requires user consent, and especially, do not block it on the grounds of privacy intrusions, because it is not a tracker.

**Comment by chris13524** [comment]
Yeah, this block makes no sense to me. We switched from recaptcha over to CoinHive captchas because they were easier for our users to complete. Authedmine.com only mines if the user explicitly opts-in. All I can see this block does is harm legitimate websites trying to do good and not use more privacy invading solutions (Google or ads).

**Comment by Khrin** [comment]
Conversation ended long ago.
